src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -116,13 +116,19 @@ class AssetsRelation(config: RelationConfig, subtreeIds: Option[List[CogniteId]]
   override def uniqueId(a: AssetsReadSchema): Long = a.id
 }
 
-object AssetsRelation extends UpsertSchema with NamedRelation {
+object AssetsRelation
+    extends UpsertSchema
+    with ReadSchema
+    with NamedRelation
+    with AbortSchema
```

Comment:
Abort naming is legacy, but a bit confusing, it seems it is actually InsertOrFail/Insert (as opposed to Upsert), not sure if it makes sense to rename in this PR or maybe later

Reply:
Think separate job. Too much stuff leftover for file-content, I don't wanna get sidetracked too much.

obviously it'd be trivial to rename here, but I think in jetfire the same naming exists so I'd need to do it there too.

Reply:
Sure Can rename to insert.